src/css/components/nav.scss

```diff
@@ -27,26 +27,31 @@
 
 .nav-link {
   color: $color-textblack;
-  font-size: $sans-s5;
-  font-weight: 400;
+  font-size: $sans-s3;
+  font-weight: 500;
   padding-bottom: .2rem;
   padding-left: 0;
   padding-top: .2rem;
+  .external-link {
+    padding-right: 1.375rem;
+    background: url('#{$static-img-path}/external-link-gray.png') right center no-repeat;
+    background: url('#{$static-img-path}/external-link-gray.svg') right center no-repeat;
+    background-size: 15px 15px;
+    }
```

Comment:
This should be indented the same as `.external-link`. The linter is VERY picky 😄